bodhi/server/tasks/approve_testing.py

```diff
@@ -78,14 +78,17 @@ def approve_update(update: Update, db: Session):
     log.info(f'{update.alias} now meets testing requirements')
     # Only send email notification about the update reaching
     # testing approval on releases composed by bodhi
-    update.comment(
-        db,
-        str(config.get('testing_approval_msg')),
-        author='bodhi',
-        email_notification=update.release.composed_by_bodhi
-    )
-    notifications.publish(update_schemas.UpdateRequirementsMetStableV1.from_dict(
-        dict(update=update)))
+    if not update.autotime or update.meets_testing_requirements:
```

Comment:
I feel that we could probably simplify the logic here. In line 76 we check if update.meets_testing_requirements is False and then return, maybe we could check if it is True instead and execute the code below. What do you think ?

Reply:
It seems if we do that `test_autotime_update_zero_day_in_testing_fail_gating_is_not_pushed` fails.